scvi/external/solo/_model.py

```diff
@@ -152,6 +153,10 @@ def train(
         validation_size: Optional[float] = None,
         batch_size: int = 128,
         plan_kwargs: Optional[dict] = None,
+        callbacks: list = [],
```

Comment:
shouldn't be necessary -- can be accessed through **kwargs

Reply:
ah sounds good